test/controllers/UploadControllerSpec.scala

```diff
@@ -312,12 +310,88 @@ class UploadControllerSpec extends FrontEndTestHelper {
       checkPageForStaticElements.checkContentOfPagesThatUseMainScala(uploadPageAsString, userType = "standard")
       checkForExpectedPageContentOnOtherUploadPages(uploadPageAsString, uploadStatus = uploadStatus)
       uploadPageAsString must include(
-        s"""      <a href="/consignment/$consignmentId/file-checks" role="button" draggable="false" class="govuk-button govuk-button--primary">
+        s"""      <a href="/consignment/$consignmentId/file-checks?uploadFailed=false" role="button" draggable="false" class="govuk-button govuk-button--primary">
            |        Continue
            |      </a>""".stripMargin
       )
     }
 
+    "render the 'upload in progress' page if a standard upload is in completedWithIssues" in {
```

Comment:
Maybe reword test slightly to: `... the 'upload in progress' error page ... has a 'completedWithIssues' status`